platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/events/lifecycle/AppLifecycleListenerLogger.kt

```diff
@@ -104,6 +106,12 @@ internal class AppLifecycleListenerLogger(
         }
     }
 
+    private fun emitMetricsForEvent(event: Lifecycle.Event) {
+        if (event == Lifecycle.Event.ON_START || event == Lifecycle.Event.ON_RESUME) {
```

Comment:
we should confirm these are indeed called for all scenarios we care about. I seem to remember these 2 in particular to be tied to having an active Activity in the app